iphone/TitaniumKit/TitaniumKit/Sources/API/KrollBridge.m

```diff
@@ -563,6 +563,25 @@ - (id)krollObjectForProxy:(id)proxy
   return result;
 }
 
+- (id)require:(KrollContext *)kroll path:(NSString *)path
```

Comment:
Should the method API need to be added back to the header file as well?

Reply:
We already do. `KrollBridge` inherits from `KrollDelegate` and this method is defined in its header.
https://github.com/appcelerator/titanium_mobile/blob/master/iphone/TitaniumKit/TitaniumKit/Sources/API/KrollBridge.h#L22
https://github.com/appcelerator/titanium_mobile/blob/master/iphone/TitaniumKit/TitaniumKit/Sources/Kroll/KrollContext.h#L17